material-overrides/assets/stylesheets/polkadot.css

```diff
@@ -1525,8 +1565,9 @@ div.mermaid {
 /* Edit this page styling */
 .actions {
```

Comment:
In the `content.html` we used to have a `<div>` that wrapped the actions, but it was removed. So you can delete these styles